veracruz-server/src/veracruz_server_tz.rs

```diff
@@ -45,34 +45,30 @@ pub mod veracruz_server_tz {
             let policy: Policy =
                 Policy::from_json(policy_json)?;
 
-            let trustzone_root_enclave_uuid = Uuid::parse_str(&TRUSTZONE_ROOT_ENCLAVE_UUID.to_string())?;
-            {
+            if !TRUSTZONE_ROOT_ENCLAVE_INITIALIZED.load(Ordering::SeqCst) {
+                let trustzone_root_enclave_uuid = Uuid::parse_str(&TRUSTZONE_ROOT_ENCLAVE_UUID.to_string())?;
```

Comment:
Is `TRUSTZONE_ROOT_ENCLAVE_UUID` not a `&'static str` in which case the conversion to String, then taking a reference, is redundant?